WordPress/gutenberg
# Fix Inconsistent Labels for Lightbox Feature

## What?
Fixes https://github.com/WordPress/gutenberg/issues/55617 

This PR ensures consistent terminology between the editor (backend) and frontend for the lightbox feature. The label "Expand on click" in the editor has been updated to "Enlarge image" for alignment with the frontend aria-labels.

## Why?
Consistency in terminology is vital for accessibility and usability. Users of assistive technology will now encounter the same terms ("Enlarge image") in both the editor and frontend, reducing potential confusion. 

## How?

- Updated the editor UI setting label from "Expand on click" to "Enlarge image."
- Verified that the aria-label for the trigger button and modal dialog in the frontend is "Enlarge image," ensuring consistency across the interface.

## Testing Instructions

1. Open a post or page.
2. Insert an image block with the lightbox feature enabled.
3. Observe that the lightbox setting in the editor now reads "Enlarge image."

## Screenshots or screencast 

## After making changes 

![enlarge_1](https://github.com/user-attachments/assets/60004049-d29e-46fc-a3c2-4ae6300fcb1f)

![enalrge_2](https://github.com/user-attachments/assets/6be79f49-5499-43df-9e32-dd56aeeb21d5)

![enlarge_3](https://github.com/user-attachments/assets/336bf227-fad3-41a0-a123-499b2749502a)
